### ISSUE TITLE
Is this project still going on?

### ISSUE THREAD

**Issue by nicholascc** [MadcowD/openbrain#41]
There are no commits, but it would be nice if we were able to read the white paper. Maybe convert it to a pdf and put it in the repo?

**Comment by ghost** [comment]
Agreed, from what it looks like though this project is as dead as Kobe...

**Comment by airzocker** [comment]
Hey,
yeah, it seems very dead, but the white paper is here is you haven't found it already:
https://github.com/mlberkeley/openbrain/blob/master/docs/backprop_free_dpg/main.pdf

**Comment by ghost** [comment]
Ah, thanks Sent from Neptune From: airzockerSent: Saturday, April 11, 2020 2:20 PMTo: mlberkeley/openbrainCc: jpie726; CommentSubject: Re: [mlberkeley/openbrain] Is this project still going on? (#41) Hey,yeah, it seems very dead, but the white paper is here is you haven't found it already:https://github.com/mlberkeley/openbrain/blob/master/docs/backprop_free_dpg/main.pdf—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by MadcowD** [comment]
Hey there, main author here. Project was put down for a long while, but I'm happy to discuss if anyone's interested. Hope everyone's doing okay with the pandemic out there.

**Comment by ghost** [comment]
Same, hope you’re doing well!From: William GussSent: Saturday, April 11, 2020 7:57 PMTo: mlberkeley/openbrainCc: jpie726; CommentSubject: Re: [mlberkeley/openbrain] Is this project still going on? (#41) Hey there, main author here. Project was put down for a long while, but I'm happy to discuss if anyone's interested. Hope everyone's doing okay with the pandemic out there.—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by dekahaedra** [comment]
This project looks pretty cool, why did it stop?

**Comment by airzocker** [comment]
@MadcowD i have basically just one question right now, did the AI see the GUI/Hotbar as well or did you use the normal malmö VideoProducer? because if it did I would be very interrested in how you got an image with all the GUI/Hotbar stuff...

**Comment by ghost** [comment]
I’m assuming it was because of either low motivation or the team moving on to other things, I know that happens to me a lot.From: Domas BuracasSent: Monday, April 13, 2020 2:49 PMTo: mlberkeley/openbrainCc: jpie726; CommentSubject: Re: [mlberkeley/openbrain] Is this project still going on? (#41) This project looks pretty cool, why did it stop?—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.
